docs/examples/cookbook/drag-and-drop.md

```diff
@@ -401,7 +401,7 @@ it reparents itself to `Dragging.OverlayFrame`, so it can be seen in front of
 other UI.
 
 ```Lua
-		Parent = scope:Computed(function(use)
+		Parent = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		Parent = scope:Computed(function(use)
```

Reply:
reverted